<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="poseHLayout">
        <item>
         <widget class="QCheckBox" name="poseCheckBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Pose Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zeroPoseBtn">
          <property name="text">
           <string>Set Zero</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="poseABtn">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
          <property name="tickInterval">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="poseBBtn">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="attrLcd">
          <property name="smallDecimalPoint">
           <bool>false</bool>
          </property>
          <property name="numDigits">
           <number>7</number>
          </property>
          <property name="mode">
           <enum>QLCDNumber::Dec</enum>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zeroBtn">
          <property name="text">
           <string>Zero</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="keyBtn">
          <property name="text">
           <string>Key Attribute</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="muteBtn">
          <property name="text">
           <string>Mute: Off</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="animCurvesBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string extracomment="Select SDK anim curves"/>
          </property>
          <property name="text">
           <string>AnimCurves</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="controlsBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string extracomment="Select controls connected to this SDK pose"/>
          </property>
          <property name="text">
           <string>Transforms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string extracomment="Break connections"/>
          </property>
          <property name="text">
           <string>Delete Attribute Keys</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
